doc/nrf/device_guides/working_with_nrf/nrf70/developing/constrained.rst

```diff
@@ -29,6 +29,15 @@ You can identify and select the appropriate host MCU depending on the IoT use ca
 .. note::
    The asterisk * indicates that the SoC is supported, but the use case is not recommended due to performance or memory constraints.
 
+Firmware patch optimizations
+****************************
+For resource-constrained hosts especially with low internal flash memory, the nRF70 Series firmware patches can significantly contribute to the overall memory footprint.
```

Comment:
Switch to not using "flash" to be more generic.
Up to @richabp to decide what the correct term is. I propose "on-chip non-volatile memory" instead of "internal flash memory".
This covers the MRAM case and possibly more.

Reply:
See https://github.com/nrfconnect/sdk-nrf/pull/13093#issuecomment-1827331227